pkgs/tools/misc/vmtouch/default.nix

```diff
@@ -2,14 +2,14 @@
 
 stdenv.mkDerivation rec {
   pname = "vmtouch";
-  version = "git-20150310";
+  version = "1.0.2";
```

Comment:
We don't want users to run into situation of outdated pkgs. I hope I will patch nix to "eat" such things in futute, so we will remove this part in version.